admin-dev/themes/new-theme/js/types/global.d.ts

```diff
@@ -0,0 +1,36 @@
+interface Window {
```

Comment:
Why ` .d`  in the name of the file, can you add a little comment at the top of this file to explain what's the purpose?